@css/main_style.css

```diff
@@ -28,6 +28,50 @@
 
 ************************************************************************************************************************/
 
+/* ******************************** *
+ * ROOT element                     *
+ * ------------                     *
+ * CSS Global variables declaration *
+ * ******************************** */
+
+:root {
+  /* Global page values */
+  --global-page-width: 375px;
+  --global-page-height: 750px;
+
+  /* Separator values */
+  --separator-width: var(--global-page-width);
+  --separator-height: 114px;
+
+  /* Section 1 */
+  --section_1-width: 375px;
+  --section_1-height: 588px;
+  --section_1-padding-left: 40px;
+  --section_1-padding-right: 40px;
+
+  /* Section 2 */
+  --section_2-width: 375px;
+  --section_2-height: 588px;
+  --section_2-padding-left: 40px;
+  --section_2-padding-right: 40px;
+
+  /* Workcard */
+  --workcard-width: 327px;
+  --workcard-height: 596px;
+  --workcard-padding-left: 24px;
+  --workcard-padding-right: 24px;
+
+  /* Workcard image */
+  --works-img-height: 295px;
```

Comment:
- [ ] Congratulations on your excellent work! using this approach. 👍 
- [ ] However, noticed that using a fixed width has resulted in your design being unresponsive. 
- [ ] To address this, I would suggest considering the use of percentage-based widths instead. 

- [ ] By doing so, you can ensure that your design becomes more responsive and adaptable to different screen sizes.🥱🥱